dtfs-central-api/src/services/repositories/utilisation-reports-repo.js

```diff
@@ -2,15 +2,24 @@ const sortBy = require('lodash/sortBy');
 const db = require('../../drivers/db-client');
 const { DB_COLLECTIONS } = require('../../constants/dbCollections');
 
+/**
+ * @typedef {object} AzureFileInfo
```

Comment:
At a glance, it looks like `{Object}` is used more frequently than `{object}` in the code base, and I think the preferred choice is `Object` with a capital O

Reply:
That does seem to be the way, but I'd argue that the lower-case `object` is more accurate. I detailed some reasoning for this in a previous review [here](https://github.com/UK-Export-Finance/dtfs2/pull/2314#discussion_r1394187937).